deposit/executor/Move.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "executor"
+version = "0.0.0"
+
+[dependencies]
+MoveStdlib = { git = "https://github.com/eigerco/substrate-move.git", subdir = "language/move-stdlib", rev = "main" }
+deposit = { local = "..", addr_subst = { "std" = "0x1" } }
+
+[addresses]
+std = "0x1"
```

Comment:
please elaborate how this is auto resolve after merge?

Reply:
See the [comment above](https://github.com/eigerco/substrate-move/pull/44/files#r1403217270)

Reply:
this approach with declaring `std` is the one present in all [official repo](https://github.com/move-language/move/blob/ea70797099baea64f05194a918cebd69ed02b285/language/evm/stdlib/Move.toml#L6)'s examples I've seen and it might be confusing for consumers of our solution to understand why it works like that, but ok.